Sources/StreamChat/Models/Poll.swift

```diff
@@ -68,4 +68,62 @@ public struct Poll: Equatable {
     
     /// A list of the latest votes received for each option in the poll.
     public let latestVotesByOption: [PollOption]
+
+    /// The list of the current user votes.
+    public let ownVotes: [PollVote]
+}
+
+/// Poll domain logic helpers.
+public extension Poll {
```

Comment:
At first I was thinking if this makes sense to add it to the public API but then on the other hand, these could be used by the SwiftUI SDK as well. Just thinking out loud here.

Reply:
Yes I think that these kind of things that are the same independent of UI, they should be close to the Domain Model. This is defended by Domain Driven Design principles. This make sure that the logic is consistent between the UIKit and SwiftUI SDKs. Also, it will be useful for customers using the LLC directly. THey won't need to know how to calculate the user with most votes.